bindings/C/AUD_Sound.cpp

```diff
@@ -67,6 +68,7 @@
 using namespace aud;
 
 #define AUD_CAPI_IMPLEMENTATION
+#include "AUD_Sequence.h"
```

Comment:
I assume this is necessary for `AUD_AnimateablePropertyType`? I'm not a fan of this dependence, but it's also not critical. But if that's the reason and you don't mind, we could move `AUD_AnimateablePropertyType` to AUD_Types.h.